src/common/SelectableOptionRow.js

```diff
@@ -120,8 +123,21 @@ export default function SelectableOptionRow<TItemKey: string | number>(
                   return _(message.text, message.values);
               }
             })(),
-            learnMoreUrl,
-            globalSettings,
+            learnMoreButton && {
+              url: learnMoreButton.url,
+              text: (() => {
+                const { text } = learnMoreButton;
+                switch (typeof text) {
+                  case 'undefined':
+                    return undefined;
+                  case 'string':
+                    return _(text);
+                  default:
+                    return _(text.text, text.values);
```

Comment:
This feels like something best abstracted into a central place, with an appropriate name. I see there's a couple other examples of it above, too.

It looks like we currently don't have a handy helper for doing this unpacking of LocalizableText; mainly we just do it in the implementation of ZulipTextIntl. I'll see about doing that as a followup PR.

Reply:
#5429